app/controllers/slot_ranges_controller.rb

```diff
@@ -18,13 +18,19 @@ def update
       @slot_range.slots.destroy_all
       @slot_range.update(slot_range_parameters)
     end
-    redirect_to edit_user_path(@guider), flash: { success: "Guider \"#{@guider.name}\" has been updated" }
+    redirect_to(
+      edit_user_path(@guider),
+      success: "Slot Range \"#{@slot_range.display_title}\" has been updated"
```

Comment:
Maybe it is simpler actually just to only allow editing if the schedule is due to 'start' more than 6 weeks in advance. So for example if a schedule has been created for 3 months time, then that's fine to edit.